fractal/cmd/_task.py

```diff
@@ -132,7 +132,8 @@ async def post_task(
 async def patch_task(
     client: AuthClient,
     *,
-    task_id_or_name: str,
+    task_id: Optional[int] = None,
```

Comment:
These parameter names are not consistent with those of `delete_task`.

Reply:
I suggest we switch to `id` and `name` as we do in `delete_task` (or to `_id` and `name` if you want to avoid using a keyword).